app/assets/stylesheets/components/_card_list.css.sass

```diff
@@ -26,6 +26,12 @@
       &.ng-enter-active
         animation-name: bounceInLeft
 
+.CardList-card.is-closed
```

Comment:
I think the Discussion component has styling for the discussion itself, while this is for the cards on the main page.

The `.is-closed` is representing a state of the cards, it doesn't affect the discussion itself, just like the bouncing animation.

I can update it, though. @joaojusto